src/components/Firestore/DocumentEditor/index.tsx

```diff
@@ -90,19 +88,17 @@ const NestedEditor: React.FC<{ path: string[]; rootKey?: string }> = ({
   return (
     <>
       <div style={{ display: 'flex' }}>
-        <TextField
-          readOnly
-          value={getLeafPath(path) || rootKey}
-          placeholder="Field"
-        />
+        <TextField readOnly value={getLeafKey(path)} placeholder="Field" />
         <TextField readOnly value={getFieldType(state)} placeholder="Type" />
         <TextField
-          value={state}
+          value={JSON.stringify(state)}
```

Comment:
Why `stringify`? Is this going to turn the string `foo` into `"foo"` in the editor?

Reply:
ack this is going to be addressed in a follow up CL... this is just to get something printed for now